pkgs/applications/audio/friture/default.nix

```diff
@@ -1,7 +1,7 @@
-{ lib, fetchFromGitHub, python3Packages, wrapQtAppsHook }:
+{ lib, fetchFromGitHub, python39Packages, wrapQtAppsHook }:
 
 let
-  py = python3Packages;
+  py = python39Packages;
```

Comment:
This kind of shorthanding should only be done when using python overwrites.

```suggestion
```
Please move this to top-level.